Source/Urho3D/IO/NamedPipe.h

```diff
@@ -54,8 +54,8 @@ class URHO3D_API NamedPipe : public Object, public AbstractFile
     unsigned Write(const void* data, unsigned size) override;
     /// Return whether pipe has no data available.
     bool IsEof() const override;
-    /// Return the pipe name.
-    const String& GetName() const override { return pipeName_; }
+    /// Sets the name of the pipe
```

Comment:
Let's explicitly say that it's not supported. This comment is quite confusing: as if we can actually set the name of the pipe.